packages/core/src/App/Components/Elements/NotificationMessage/notification-close-mx-mlt.jsx

```diff
@@ -4,7 +4,7 @@ import { isMobile } from '@deriv/shared';
 import { Button, Icon, Text } from '@deriv/components';
 
 const NotificationCloseMxMlt = ({ header, message, secondary_btn, img_src, img_alt, onClose }) => (
-    <div className='close-mx-mlt notification-banner'>
+    <div data-testid='notification-close-Mx-Mlt' className='close-mx-mlt notification-banner'>
```

Comment:
```suggestion
    <div data-testid='notification-close-mx-mlt' className='close-mx-mlt notification-banner'>
```
@niloo-fs i suggest not using capital letters, just kebab-case (in the respective test too)🙂

Reply:
Thank u @maryia-binary will change these